experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -622,38 +622,43 @@ describe("SharedTree", () => {
 
 		it("does not undo edits made remotely", async () => {
 			const value = "42";
-			const value2 = "43";
 			const provider = await TestTreeProvider.create(2);
 			const [tree1, tree2] = provider.trees;
 
+			const initialState: JsonableTree = {
+				type: brand("TestValue"),
+				value: "A",
+			};
+			initializeTestTree(tree2, initialState);
+			await provider.ensureSynchronized();
+
+			assert.equal(getTestValue(tree1), "A");
+
 			// Insert node
-			setTestValue(tree1, value);
+			insert(tree1, 0, value);
+			await provider.ensureSynchronized();
+
+			const testValuesAfterInsertion = getTestValues(tree1);
+			assert.equal(testValuesAfterInsertion[0], "A");
+			assert.equal(testValuesAfterInsertion[1], value);
+
 			// Make a remote edit
-			setTestValue(tree2, value2);
+			remove(tree2, 1, 1);
 			await provider.ensureSynchronized();
 
-			// Validate insertion
-			const readCursor = tree1.forest.allocateCursor();
-			moveToDetachedField(tree1.forest, readCursor);
-			assert.ok(readCursor.firstNode());
-			assert.equal(readCursor.value, value);
-			assert.ok(readCursor.nextNode());
-			assert.equal(readCursor.value, value2);
-			readCursor.free();
+			// Validate insertion and deletion
```

Comment:
The comment says `validate insertion and deletion` but is this portion of the code validating just deletion?

Reply:
Yup, good catch. This was left over :)